components/playground/playground.go

```diff
@@ -1135,9 +1259,20 @@ func (p *Playground) bootCluster(ctx context.Context, env *environment.Environme
 		}
 	}
 
-	if pdAddr := p.pds[0].Addr(); len(p.tidbs) > 0 && hasDashboard(pdAddr) {
-		fmt.Printf("TiDB Dashboard:  ")
-		colorCmd.Printf("http://%s/dashboard\n", pdAddr)
+	if len(p.dmMasters) > 0 {
+		fmt.Printf("Connect DM:      ")
+		endpoints := make([]string, 0, len(p.dmMasters))
+		for _, dmMaster := range p.dmMasters {
+			endpoints = append(endpoints, dmMaster.Addr())
+		}
+		colorCmd.Printf("tiup dmctl --master-addr %s\n", strings.Join(endpoints, ","))
+	}
+
+	if len(p.pds) > 0 {
```

Comment:
why mention pd here?

Reply:
Now we can start a dm cluster without pd, so allow pd to be zero
```
tiup-playground nightly --dm-master 1 --dm-worker 1 --db 0 --kv 0 --pd 0 --tiflash 0 
```